{
    "functionName": "test_wrong_exception",
    "className": "TestUpdateError",
    "fileName": "/poletaevvlad_&_CubeLang/test_&_execution_&_test_runtime_errors.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "e = NameError()\nwith pytest.raises(NameError):\n    RuntimeError.update_error('function', e)\n"
}